packages/app-explorer/src/index.css

```diff
@@ -55,6 +55,10 @@
       white-space: nowrap;
     }
   }
+
+  .value {
+    font-family: monospace;
```

Comment:
The logs aren't decoded properly here, it should have the actual values, i.e. the same as transactions/events. So monospace is not correct, neither is the actual dropdown above. (The wrapping however is)